trulens_eval/trulens_eval/tru_virtual.py

```diff
@@ -407,12 +407,11 @@ def __init__(
         if (end_time - start_time).total_seconds() == 0.0:
             end_time += datetime.timedelta(microseconds=1)
 
-        if "cost" not in kwargs:
```

Comment:
seems cost and perf were out of __init__'s kwargs into a named argument but this was not been updated